config/environments/production.rb

```diff
@@ -83,6 +83,9 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
+  # Disable ActiveRecord logging
+  ActiveRecord::Base.logger = Logger.new('/dev/null')
```

Comment:
Hi @ngw, could we just set `config.log_level` to `info` or `warn` to prevent this egress, rather than losing all AR logging (including any errors)